### DISCUSSION TITLE
Need ability to control order of scenario execution inside a spec

### DISCUSSION THREAD

**Discussion by deepesh1511** [getgauge/gauge#2322] [Ideas]
We are migrating from Selenium to Gauge framework for UI and API automation in our project and need the ability to control the order of scenario execution inside a spec. 

Is there any functionality similar to TestNG prioritization? Our existing tests heavily depend on the priority annotation and would like to know if anything similar can be implemented in Gauge.

**Comment by zabil** [comment]
Unfortunately scenarios cannot be assigned a priority or assigned an order. So it's best to write them in a way that they can run independently.

**Reply by deepesh1511** [reply]
@zabil 
What could be the possible work around to handle such scenarios?

**Reply by zabil** [reply]
There is no workaround. There is a discussion around this at https://github.com/getgauge/gauge/issues/1991 but again no solutions.

**Comment by DeLaphante** [comment]
Struggling to design or build an advanced, stable and well structured test automation framework?

Check out & play around with my powerful and robust Selenium Web UI & API test automation framework on Github -> https://github.com/DeLaphante/CynkyAutomation

Save yourself some time and use this ready made framework for any project and get the technical support you need to succeed!
